toolkit/tools/scheduler/schedulerutils/buildlist.go

```diff
@@ -123,3 +126,15 @@ func ReadReservedFilesList(path string) (reservedFiles []string, err error) {
 
 	return reservedFiles, nil
 }
+
+// rpmsToFind should be an absolute path to the expected RPM, while toolchainRPMs is a list of '<name>-<version>-<release>.rpm'.
+func IsToolchainRPM(rpmPath string, reservedRPMs []string) bool {
```

Comment:
All exported functions should have a documentation string before their definition.

Reply:
Added